repo2docker/buildpacks/python/__init__.py

```diff
@@ -1,39 +1,80 @@
-"""
-Generates a variety of Dockerfiles based on an input matrix
-"""
+"""Generates Dockerfiles based on an input matrix based on Python."""
 import os
+
 from ..base import BaseImage
 
 
 class PythonBuildPack(BaseImage):
+    """Setup Python 3 for use with a repository."""
     def get_packages(self):
+        """Return list of Python3 packages to be installed."""
```

Comment:
"Return list of system-level Python3 dependencies" (I don't like that sentence that much). Something to clarify that these are the apt-get packages we install, not python packages (like say numpy).

Reply:
Good distinction @betatim. Will update.